core/src/main/java/com/google/zxing/pdf417/decoder/DecodedBitStreamParser.java

```diff
@@ -155,8 +164,8 @@ static DecoderResult decode(int[] codewords, String ecLevel) throws FormatExcept
     return decoderResult;
   }
 
-  private static int decodeMacroBlock(int[] codewords, int codeIndex, PDF417ResultMetadata resultMetadata)
-      throws FormatException {
+  protected static int decodeMacroBlock(int[] codewords, int codeIndex, PDF417ResultMetadata resultMetadata)
```

Comment:
Last nit: if this is just visible for testing, it can probably be package-private only? 
I had also done that extra 4-space indent for wrapping continuations like the 'throws' clause on purpose, but, I don't really care.

Reply:
sure no problem; will change the visibility

I'm lazy so I just use the auto layout from IDEA, which works usually quite well, but doesn't indent like that, there are additional changes like that, shall I change them back as well?

Reply:
If you're editing this line anyway, maybe change it back, but wouldn't worry about undoing every whitespace change.

Reply:
I changed it back mostly, makes the change a bit cleaner, shall i rebase or will you do a squash merge?